### DISCUSSION TITLE
Aprender Design sem faculdade

### DISCUSSION THREAD

**Discussion by phenriqueleao** [frontendbr/forum#1419] [Discussão]
Olá, pessoal!

Vejo muito sobre como ser um(a) programador(a) autodidata, porém vejo que é muito mais difícil de encontrar material para se estudar design gráfico.

Curso Sistemas para Internet e gosto bastante de front-end, porém sempre preciso apelar à interfaces criadas por algum designer já que não tenho a expertise necessária.

Pensando nisso e vendo que existem vários repositórios "self taught programmer", gostaria de saber que passos eu poderia seguir para aprender design de forma autodidata também.

Grato pela atenção!

**Comment by NicholasKuchiniski** [comment]
> Curso Sistemas para Internet e gosto bastante de front-end, porém sempre preciso apelar à interfaces criadas por algum designer já que não tenho a expertise necessária.

Quem tem que saber "criar design" é o designer, não o desenvolvedor, eu não me preocuparia com isso.

**Comment by willycamargo** [comment]
Ao contrário do @NicholasKuchiniski  eu acho que é muito importante sim aprender sobre conceitos teóricos e práticos de Design e UX. E apesar de achar irrelevante para a thread argumentar sobre isso, só gostaria de destacar que esse "diferencial" já foi responsável por muitas conquistas profissionais.

___

Sobre os passos, não sei dizer exatamente qual o melhor processo para aprender sobre design, mas vou falar um pouco sobre minha experiência e como eu tento estar melhorando e aprendendo sobre Design e UX no dia a dia.

* Estou sempre em contato com meus colegas de trabalho que são responsáveis pela arquitetura e design da aplicação e sempre perguntando e tentando entender o porque das coisas serem como são.

* Consumo artigos, vídeos e podcasts sobre o assunto (newsletters são ótimas para isso)

* Quando não existe um design feito eu nunca utilizo frameworks CSS com interfaces visuais prontas. Nem que eu precise me basear em interfaces já existentes, eu prefiro criar meus componentes do zero. Aprendi muito fazendo isso. Composição de cores, bordas, espaçamentos, sombras etc. Por exemplo: era comum em alguns momentos perceber que o componente que criei "está faltando algo" e, quando vejo com calma, percebo que é uma borda ou sombra bem delicada que faz toda a diferença. Com o tempo o senso crítico e a visão aguçada vão melhorando.

Além desses itens acima, com o tempo fui aprendendo a refletir sobre como o design é importante em nosso dia a dia, como no formato de um mouse, teclado, copo, livro, carteira, bolsa etc.
Esse tipo de curiosidade muitas vezes me leva a pesquisar e entender mais, principalmente sobre história e arte. Me gera insights interessantes, traçando uma comparação em como as interfaces vêm se modificando para atender as necessidades das pessoas de forma cada vez mais simples.

Também vale lembrar que quando falamos sobre "criatividade", [Tudo é uma mistura de referências](https://www.everythingisaremix.info). Quando você diz que "[...]já que não tenho a expertise necessária", acredito que o necessário para criar interfaces seria:

* Bom conhecimento técnico em HTML e CSS
* Consumir conteúdos e interfaces suficientes para montar seu "banco de dados de referências" necessário para copiar, transformar e combinar.
_____

Dificilmente iremos ser tão bons e tão produtivos quanto profissionais que trabalham especificamente com isso, mas ao passar do tempo o conhecimento cada vez mais aprofundado em design, usabilidade e acessibilidade podem ser um diferencial importante para sua carreira profissional. Parabéns pelo interesse e espero que outras pessoas tenham relatos melhores!

**Comment by onlurking** [comment]
Fala @phenriqueleao, me encontro na mesma situação que você! Eu topei com algumas referências ao longo do tempo, aqui estão elas:

- [This is a web page](https://justinjackson.ca/words.html), é um balde de água fria quando o assunto é **overdesign**.
- [Web Design in 4 minutes](https://jgthms.com/web-design-in-4-minutes) do criador do [Bulma](https://bulma.io), é uma introdução bem interessante de como o design pode colocar conteúdo em contraste.
- [Crash Course: Design for Startups](https://paulstamatiou.com/startup-web-design-ux-crash-course), aqui tem ótimas recomendações e referências(eg. o livro [Don't Make Me Think](https://www.amazon.com.br/Dont-Make-Think-Revisited-Usability/dp/0321965515) que basicamente dita que bom design é um design invisível e intuitivo, e o site [52 Weeks of UX](https://52weeksofux.com/tagged/week_1) com vários artigos, destancando o [The Five W’s of UX](https://52weeksofux.com/post/890288783/the-five-ws-of-ux)).
- [Design - Zen of Programming](https://zen-of-programming.com/design), um post mais prático sobre design na Web, tem umas coisas bem interessantes como esse slide sobre um [redesign fícticio do Linkedin](https://speakerdeck.com/sschoger/the-little-details-of-ui-design).
- [UX Crash Course: 31 Fundamentals](https://thehipperelement.com/post/75476711614/ux-crash-course-31-fundamentals), outro site alá "52 weeks of UX", não cheguei a ver ainda, mas está na minha lista, ele também tem uma continuação: [UX Crash Course: User Psychology](https://thehipperelement.com/post/87574750438/ux-crash-course-user-psychology).
- Sempre é bom lembrar que [Web Design is 95% Typography](https://ia.net/topics/the-web-is-all-about-typography-period), outro link interessante sobre tipografia: [A Five-Minute Guide to Better Typography](https://www.pierrickcalvez.com/journal/a-five-minutes-guide-to-better-typography)

Não tem haver com Web.. entrentanto eu recomendo dar uma olhada no [Human Interface Guidelines](https://elementary.io/docs/human-interface-guidelines#human-interface-guidelines) do ElementaryOS, tem uns insights muito legais sobre UX no geral.

**Outros links:**
- [Awesome UX](https://github.com/netoguimaraes/awesome-ux) lista cheio de recursos de UX/UI.
- [HackDesign](https://hackdesign.org/lessons101), uma newsletter de design cheia de links interessantes no acervo.
- [User Experience - Springboard](https://www.springboard.com/resources/learning-paths/user-experience-design), é um curso grátis de UX, eu não fiz ele ainda.

**Comment by klimbo123** [comment]
Quem tem que saber "criar design" é o designer, não o desenvolvedor
[9Apps](https://9apps.ooo/) [Showbox](https://showbox.kim/) [Tutuapp](https://tutuapp.uno/)

**Comment by willycamargo** [comment]
> Quem tem que saber "criar design" é o designer, não o desenvolvedor

Aparentemente uma conta fake criada há 5 minutos atrás apenas com intenção de causar no forum.
Bem vindo.

Poderia definir o que seria "criar design" e esclarecer com argumentos sólidos de por que isso é uma atividade que apenas designer deve saber e não desenvolvedores?

**Comment by Edvan27** [comment]
Sei que cheguei um pouco atrasado mas costumo acompanhar um [canal do Youtube](https://www.youtube.com/channel/UCw-CQYphBIs_nplLNJuz44w) que posta varias dicas legais.

**Comment by kvnol** [comment]
Eu aprendi design sem faculdade, é difícil, mas tem sim muito conteúdo de design na internet. Falando pouco e deixando muito conteúdo, se liga:

https://github.com/gztchan/awesome-design

**Comment by phenriqueleao** [comment]
Gostaria de agradecer imensamente todos que compartilharam suas fontes na issue. Vi que tem muito material legal e já é um baita norte, era isso mesmo que estava procurando.

Muito obrigado!

Edit: vou deixar a issue aberta para caso alguém mais resolva compartilhar mais material.

**Comment by salatielsql** [comment]
Oi @phenriqueleao sempre trabalhei como designer e front-end ao mesmo tempo, mas a 3 meses decidir foca só em front-end estudando mais o JS, que era meu lado fraco do front. Acho bem interessante desenvolvedores terem noções de design.

A um tempo criei um site que é um repositório de conteúdo para quem quer aprender ux/ui design:  https://uxverso.com.br/

Mas acredito que um bom início seria ler o livro Não me faça pensar ele vai te dar uma boa visão sobre usabilidade web. Qualquer dúvida só chamar que podemos conversar :D
